app/main.py

```
@@ -36,89 +124,80 @@ def process_csv(file_content: bytes, filename: str) -> None:
-@app.get("/users/{user_id}", response_model=UserResponse)
-def get_user(user_id: int, db: Session = Depends(get_db)) -> UserResponse:
-    """
-    Retrieves a user by their ID.
-    
-    Args:
-        user_id (int): The ID of the user to retrieve.
-
-    Raises:
-        HTTPException: If the user is not found.
+@app.get("/user/{user_id}/", response_model=UserResponse)
+async def get_user(user_id: int, db: AsyncSession = Depends(get_db)) -> UserResponse:
```

Ruff

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)